package.json

```diff
@@ -38,6 +38,7 @@
     "rxjs": "6.0.0",
     "systemjs": "0.19.43",
     "tsickle": "^0.27.2",
+    "parse5": "^5.0.0",
```

Comment:
This won't solve the problem since this doesn't do anything for downstream apps. For people using schematics, they would need to have parse5 installed in _their_ app. However, we don't want to include that as a real dependency in the distributed package.json. Maybe that's motivation to use something else?

Reply:
Gotcha, but shouldn't we be including this anyway? Its just out of chance this is working.

Reply:
Yes, but I assumed the main goal here was to fix #11341 (which I assumed is talking about downstream apps)

Reply:
Ya, that was the original goal but as you said it won't solve the problem. Let me know how you would like to proceed.

Reply:
@brocco @filipesilva do you have any thoughts on how we could do HTML parsing without making parse5 a `peerDep` of `@angular/material`?

Reply:
Hmm not really sure. If this package uses `parse5` then it should be a dependency. But I see the problem in adding when it's just used for the update schematics.

I suppose one solution is to separate the Material schematics into a separate package. Another is to provide html parsing facilities within schematics. But one way or another, Material would have `parse5` somewhere in its dependency (or peer dependency) tree.

Reply:
The little bit of HTML updating we've done is via regex, but we're also only adding some tags inside of the head tag for PWA.